polkadot/xcm/xcm-executor/src/traits/asset_transfer.rs

```diff
@@ -23,8 +23,6 @@ use xcm::prelude::*;
 /// Errors related to determining asset transfer support.
 #[derive(Copy, Clone, Encode, Decode, Eq, PartialEq, Debug, TypeInfo)]
 pub enum Error {
-	/// Invalid non-concrete asset.
-	NotConcrete,
 	/// Reserve chain could not be determined for assets.
 	UnknownReserve,
```

Comment:
Is this part of the XCM standard @franciscoaguirre?  
Then the following variant would need to get explicit index 1, to not break compatibility.

Reply:
I added the explicit index 1 in this commit: https://github.com/paritytech/polkadot-sdk/pull/3867/commits/37f4e30962fedff3ca4f3b8a1029b1187a098a63

Reply:
It's not a part of the XCM standard, it's only used in pallet-xcm for determining whether a teleport or a reserve asset transfer needs to be done. So we can remove this explicit index.
Also, I'd add a comment about this, probably on the module itself.